ruby/cross_service_examples/item-tracker/README.md

```diff
@@ -0,0 +1,100 @@
+
+## 1. Front-end
+The front-end code is written in JavaScript and implemented using the React framework.
+For more information, see the [React Elwing Client README](../../../resources/clients/react/elwing/README.md).
+
+## 2. Back-end
+The centerpiece of this example is an API written exclusively in Ruby. It features the following gems:
+* [AWS SDK for Ruby](https://aws.amazon.com/sdk-for-ruby/) (for communicating with Amazon Aurora)
+* [Sinatra](https://sinatrarb.com/intro.html) (for a lightweight API implementation)
+* [Sequel](https://sequel.jeremyevans.net/) (for Ruby-native modeling of SQL queries)
+
+This Ruby API consists of the following API routes:
+
+|method              | route                |action        | client             |function                |
+|--------------------|----------------------|--------------|--------------------|------------------------|
+|GET                 | /api/items           |List items    | RDSDataService     |execute_statement(*SQL*)|
+|POST                | /api/items           |Add item      | RDSDataService     |execute_statement(*SQL*)|
+|PUT                 | /api/items:archive   |Archive item  | RDSDataService     |execute_statement(*SQL*)|
+|GET                 | /api/items/{item_id} |Get item      | RDSDataService     |execute_statement(*SQL*)|
+|POST                | /api/items:report    |Create report | RDSDataService     |execute_statement(*SQL*)|
+
+## 3. Database
+This example relies on an MySQL 5.7 database implemented using Amazon Aurora, a serverless relational database management system (RDBMS). 
+The Aurora cluster is deployed using the AWS Cloud Development Kit (AWS CDK). 
+For more information, see the [Aurora Serverless App CDK script README.md](../../../resources/cdk/aurora_serverless_app/README.md).
+
+***
+
+# Invoke this example code
+
+## Pre-requisites
+To explore this example, you must first:
+1. Create account & configure credentials using [instructions in the top-level README.md]((../../../README.md#invoke-example-code)).
+2. Install Ruby and resolve dependencies using [instructions in the Ruby README.md](../../../ruby/README.md).
+3. Create AWS infrastructure using [instructions in the Aurora Serverless App README.md](../../../resources/cdk/aurora_serverless_app/README.md). 
+    * NOTE: Store the following output values in the `helpers/config.yml` file: `CLUSTER_ARN`, `SECRET_ARN`, and `DATABASE`. Please store these in the `helpers/config.yml` file.
+4. Populate your database with a table and data using the following commands:
+     ```bash
+     cd ruby/cross_service_examples/item-tracker/helpers
+     ruby create_table.rb # checks for database and creates a new table
+     ruby populate_data.rb # creates a table with 10 records
+     ```
+5. Register your email address with Amazon Simple Email Service (SES) using the [instructions on the AWS SES docs](https://docs.aws.amazon.com/ses/latest/dg/creating-identities.html)
```

Comment:
Use the title of the link destination for the link text:
instructions on the AWS SES docs ->  Creating and verifying identities in Amazon SES